blogp3.md

```diff
@@ -0,0 +1,155 @@
+                "waitingtime": {
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        }
+    },
+    "type": "object"
+}
+````
+7.
+![](https://github.com/11301858/callcenterapp/blob/master/tutorial/p3images/29.png)
+
+Output your processing data into another container. 
+
+Use the condition within the for-each loop to do whatever data processing you want with your data. 
+To send your processed data you could do something like this:
+
+![](https://github.com/11301858/callcenterapp/blob/master/tutorial/p3images/30.png)
+
+Click “Save” and “Run” in the top menu of the logic app. The app won’t do anything until you upload something in the blob you specified. 
+
+![](https://github.com/11301858/callcenterapp/blob/master/tutorial/p3images/31.png)
+ 
+Go to your blob. You can manually upload some json files that are in line with the previously detailed schema, or use the following C code if you have a C compiler (clang, gcc, etc.). 
+```` C
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#define FILES 50
+#define SIZE sizeof(int) + 1
+
+int choice3 (int);
+
+int main() {
+  char yesno [2][4] = {{"Yes"}, {"No"}};
+  char problemarea [5][11] = {{"Problem 1"}, {"Problem 2"}, {"Problem 3"}, {"Problem 4"}, {"Other"}};
+  char callcenter [5][13] = {{"CallCenter 1"}, {"Callcenter 2"}, {"Callcenter 3"}, {"Callcenter 4"}, {"Other"}};
+  char lastname [6][6] = {{"Mitra"}, {"Sam"}, {"Tom"}, {"Ann"}, {"Mary"}, {"Todd"}};
+  char comment [4][22] = {{"The call went great!"}, {"The call was boring."}, {"The call was annoying."}, {"The call was bad."}};
+
+  FILE *fp;
+  char prefix [10] = {"data"};
+
+    for (int file_index = 0; file_index < FILES; file_index++){
+      char suffix [SIZE + 5];
+      sprintf(suffix, "%d", file_index);
+      strcat(suffix, ".txt");
+      strcat(prefix, suffix);
+      printf("Now creating ");
+      printf("%s\n", prefix);
+      fp = fopen(prefix, "w+");
+      fprintf(fp,"{\n \"formdata\": {\n\t\"blocked\": \"%s\",\n\t\"abandoned\": \"%s\",\n\t\"waitingtime\": \"%d\",\n\t\"problemarea\": \"%s\",\n\t\"issueresolved\": \"%s\",\n\t\"resolvetime\": \"%d\",\n\t\"calltime\": \"%d\",\n\t\"aftercallwork\": \"%d\",\n\t\"lname\": \"%s\",\n\t\"country\": \"%s\",\n\t\"subject\": \"%s\",\n},\n}",
+      (yesno[choice3(2)]),
+      (yesno[choice3(2)]),
+      1 + rand()%10,
+      (problemarea[choice3(5)]),
+      (yesno[choice3(2)]),
+      1 + rand()%10,
+      1 + rand()%10,
+      1 + rand()%10,
+      (lastname[choice3(6)]),
+      (callcenter[choice3(6)]),
+      (comment[choice3(4)])
+    );
+      fclose(fp);
+      strcpy(prefix, "data");
+
+    }
+
+}
+
+int choice3 (int size){
+  srand(time(0));
+  return rand()%(size-1);
+}
+````
+
+The code will create folders in your local directory. Upload these folders to the Azure Blob Storage container you are using to receive the form data. 
+
+See the results, enjoy, and customize to your needs!
```

Comment:
add a screenshot of the intended output and visual state of the system. you can show them the results instead of just saying "see the results"

Reply:
Done